src/Grpc/test/InteropTests/InteropTests.csproj

```diff
@@ -25,8 +25,8 @@
   <!-- Shared framework definitely exists when this target runs. Test asset builds require shared Fx. -->
   <Target Name="PublishTestAssets" BeforeTargets="Publish;RunTests;Test">
     <ItemGroup>
-      <_ProjectsToPublish Include="..\testassets\**\*.csproj"
-          AdditionalProperties="PublishDir=$(PublishDir)%(Filename)" />
+      <_ProjectsToPublish Include="..\testassets\**\*.csproj" />
+      <_ProjectsToPublish AdditionalProperties="PublishDir=$(PublishDir)%(Filename)" />
```

Comment:
@rainersigwald why doesn't metadata work correctly when creating an item group using wildcards❔ Things seem to always work when including from another item group or updating one that exists (like the "fixed" case).

Is there any way to set this item group in one go❔